h3/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #[deny(missing_docs)]
 pub mod quic;
 
+mod frame;
```

Comment:
Maybe we should silence all the "unused" warnings with temporary `#[allow(dead_code)]`. Then removed them once they are effectively used through public API.